projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -14,7 +14,7 @@ export const backends = [
     {
         title: 'PROD proofed',
         value: 'https://editions-proof-prod.s3-eu-west-1.amazonaws.com/',
-        preview: false,
+        preview: true,
```

Comment:
presuming it will turn on only for gu members?

Reply:
yep they need to have access to the duck menu to change their endpoint